### ISSUE TITLE
Torch batchnorm issue with Aimet 1.32.2

### ISSUE THREAD

**Issue by shayaanjamil-10xe** [quic/aimet#3152]
Getting this error when I am trying to train RTM-det with QAT.
![image](https://github.com/quic/aimet/assets/172261954/d3663875-f1ba-40f8-94f4-6e75b0a4916f)

Steps I took to prepare for QAT:
![image](https://github.com/quic/aimet/assets/172261954/7242f6b6-35fe-4a9a-a94b-e2dae3892dc9)

**My package versions**
aimet-torch               1.32.2
torch                     2.2.2
with cuda 12.0

**Comment by quic-bhushans** [comment]
Hi @shayaanjamil-10xe please do self.model.eval() before calling prepare_model()

also please use latest aimet-torch from here which has improved support https://pypi.org/project/aimet-torch/
